README.md

```diff
@@ -182,6 +182,61 @@ tell it to do.
 
 Enjoy streaming Google Music via Alexa!
 
+### (Optional) Setting up a Heroku instance
+
+Setting up an instance on Heroku may be an easier option for you, and these
+instructions detail how to accomplish this. The following steps replace the need
+to setup a local server. First one must have Heroku setup on your local machine and
+an account associated. Go to https://heroku.com for information about this.
```

Comment:
Change the last sentence to `Visit [the CLI documentation](https://devcenter.heroku.com/articles/heroku-cli) for details on setting this up`.